portfolio/src/main/webapp/leave-comment/leave-comment.html

```diff
@@ -12,18 +12,19 @@
   <body>
     <div id="leave-comment-form">
       Please share your feedback!
-      <form>
-        <input type="email" placeholder="Your email adress">
+      <form method="post" action="/add-comment">
+        <input name="email" type="email" placeholder="Your email adress">
         <textarea name="text" wrap="soft" placeholder="Your comment"></textarea>
         <input class="button" type="submit" placeholder="Submit!">
       </form>
     </div>
     <div id="sidebar">
       <div id="comment-list">
-        Here the comments will be displayed.
+        Comments:
       </div>
       <a class="button" href="../index.html">Main page</a>
     </div>
-    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+    <script src="script.js"></script>
```

Comment:
Putting `<script>` tags at the end of the HTML holds off rendering the page until the `.js` file is loaded.

Reply:
I didn't know async and defer, thank you! 😀

Reply:
I have same problem in another html file (travels), but I don't have clean solution for that file (I'll think how to repair it). For now, I mark this one as done.